src/common-utils/scripts/bash_theme_snippet.sh

```diff
@@ -23,3 +22,20 @@ __bash_prompt() {
 }
 __bash_prompt
 export PROMPT_DIRTRIM=4
+
+# Function to set the terminal title to the command being executed
+preexec() {
+    local cmd=$(history 1 | sed 's/^[ ]*[0-9]*[ ]*//')
```

Comment:
Should we check if `history` command works?  The `history 1` command might not work as expected in all contexts, especially if the history is not enabled or if the shell history settings are different.

Reply:
Changed it so we use `BASH_COMMAND` instead